compiler-rt/lib/radsan/radsan_context.cpp

```diff
@@ -62,6 +64,8 @@ void Context::printDiagnostics(const char *intercepted_function_name) {
           "Real-time violation: intercepted call to real-time unsafe function "
           "`%s` in real-time context! Stack trace:\n",
           intercepted_function_name);
+
+  atomic_fetch_add(&radsan::radsan_report_count, 1, memory_order_relaxed);
```

Comment:
The base of the "exit non-zero in continue mode". This is what msan does. 

In a future review, we then hook into the `exit` call and check this number.